images/tcp-stream.proto

```diff
@@ -16,9 +25,6 @@ message tcp_stream_entry {
 	optional uint32		rcv_wscale	= 8;
 	optional uint32		timestamp	= 9;
 
-	optional bool		cork		= 10;
-	optional bool		nodelay		= 11;
```

Comment:
Same here, we should leave those lines in the image description, so that new criu can read older images.